xql/main.py

```diff
@@ -0,0 +1,135 @@
+
```

Comment:
Add license information at the top of all files.

Reply:
Added.